.rspec

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
+--format documentation
```

Comment:
Set up your text editor to always add newline at the end of files, please

Reply:
It's important that all of your files have a newline at the end. Please add it to files which lack them, and configure vs_code to do that automatically.